scripts/build-tree-sitter.sh

```diff
@@ -7,48 +7,50 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 TMP_BUILD_DIR=$( mktemp -d )
 mkdir $TMP_BUILD_DIR/build
 
-PKG_CONFIG_PATH="$TMP_BUILD_DIR/build/lib/pkgconfig"
-export PKG_CONFIG_PATH=$(pkg-config --variable pc_path pkg-config)${PKG_CONFIG_PATH:+:}${PKG_CONFIG_PATH}
```

Comment:
this suppose to be:

```
export PKG_CONFIG_PATH=${PKG_CONFIG_PATH:+:}${PKG_CONFIG_PATH}:$(pkg-config --variable pc_path pkg-config)
```

then it will pickup correct `.pc` files